src/app/league/models.py

```diff
@@ -30,17 +33,28 @@ def build_key(cls, league_id, user_key):
         return key
 
     def update_participant_count(self, size):
-        self.participant_count = size + self.participant_count
+        self.participant_count += size
         self.put()
 
 
-def create_league(user, name, rating_scheme, description=None):
+def create_league(user, name, rating_scheme, k_factor_scaling, description=None):
     if rating_scheme not in LeagueModel.scheme_choices:
         raise InvalidRatingSchemeException("Invalid rating scheme %s" % rating_scheme)
     league_id = LeagueModel.generate_id()
     key = LeagueModel.build_key(league_id, user.key)
-    new_league = LeagueModel(key=key, league_id=league_id, name=name, rating_scheme=rating_scheme,
-                             description=description)
+
+    if rating_scheme is LeagueModel.scheme_choices[0]:
```

Comment:
rating_scheme is the mathematical model we use (right now ELO is the only real one), why check the rating scheme to determine k_factors?